.bazelrc

```diff
@@ -0,0 +1 @@
+common --incompatible_use_python_toolchains=false
```

Comment:
Maybe let's add a comment referencing the issue that requires this?

Reply:
Done.